website/datafuse/docs/development/contributing.md

```diff
@@ -18,6 +18,8 @@ Thank you.
 
 ## Contributing
 
+Note: In order to run `make test` successfully, make sure you have installed `mysql` client.
```

Comment:
The step `make setup` will install mysql client:
https://github.com/datafuselabs/datafuse/blob/master/scripts/setup/dev_setup.sh#L326

Reply:
I think we should make the `OPERATIONS` to true by default in dev_setup.sh

Reply:
`if [[ "$INSTALL_BUILD_TOOLS" == "false" ]] && \
   [[ "$OPERATIONS" == "false" ]] && \
   [[ "$INSTALL_PROFILE" == "false" ]] && \
   [[ "$INSTALL_PROVER" == "false" ]] && \
   [[ "$INSTALL_CODEGEN" == "false" ]]; then
   INSTALL_BUILD_TOOLS="true"
fi`

change to true will break this?

Reply:
There are two ways for the dev_setup.sh to install MySQL client:
1. set `OPERATIONS` to true by default in dev_setup.sh
2. Makefile
setup:
        bash ./scripts/setup/dev_setup.sh -o

Reply:
set OPERATIONS to true by default in dev_setup.sh will cause INSTALL_BUILD_TOOLS="false", it will not install build tools,right?

Reply:
> set OPERATIONS to true by default in dev_setup.sh will cause INSTALL_BUILD_TOOLS="false", it will not install build tools,right?
Y.
I have check the codes dev_setup.sh, it seems `OPERATIONS` is not make sense here, MySQL client is a tool, I am not sure why we use `OPERATIONS` to enable it.
@ZhiHanZ may help here

Reply:
Hi @GrapeBaBa 

This issued here https://github.com/datafuselabs/datafuse/issues/1650

Shall we close the PR first?
Thank you for the contribution and suggestion.